source/wp-content/themes/wporg-parent-2021/patterns/post-navigation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Post Navigation
+ * Slug: wporg-parent-2021/post-navigation
+ * Inserter: no
+ */
+
+?>
+
+<!-- wp:group {"style":{"border":{"top":{"color":"var:preset|color|light-grey-1","width":"1px"}},"spacing":{"padding":{"top":"var:preset|spacing|20","right":"var:preset|spacing|20","bottom":"var:preset|spacing|20","left":"var:preset|spacing|20"}}},"layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between"}} -->
+<div
+	class="wp-block-group"
+	style="
+		border-top-color: var( --wp--preset--color--light-grey-1 );
+		border-top-width: 1px;
+		padding-top: var( --wp--preset--spacing--20 );
+		padding-right: var( --wp--preset--spacing--20 );
+		padding-bottom: var( --wp--preset--spacing--20 );
+		padding-left: var( --wp--preset--spacing--20 );
+	"
+>
```

Comment:
This auto-formatting unfortunately breaks the block validation when loaded in the editor, and while it doesn't have the same issue in `wp:pattern` blocks it's better to be safe.

```suggestion
<div class="wp-block-group" style="border-top-color:var(--wp--preset--color--light-grey-1);border-top-width:1px;padding-top:var(--wp--preset--spacing--20);padding-right:var(--wp--preset--spacing--20);padding-bottom:var(--wp--preset--spacing--20);padding-left:var(--wp--preset--spacing--20)">
```

I'm also guessing this is the inline CSS you mentioned copying? This is fine, it's part of the block itself. Removing it would also cause validation errors (and wouldn't load on the frontend — since this is a static block, the content here is what's used, the attributes in the JSON comment are only used by the editor).